controllers/drplacementcontrol.go

```diff
@@ -1048,6 +1048,9 @@ func (d *DRPCInstance) relocate(preferredCluster, preferredClusterNamespace stri
 	d.setDRState(drState)
 	addOrUpdateCondition(&d.instance.Status.Conditions, rmn.ConditionAvailable, d.instance.Generation,
 		d.getConditionStatusForTypeAvailable(), string(d.instance.Status.Phase), "Starting relocation")
+	addOrUpdateCondition(&d.instance.Status.Conditions, rmn.ConditionPeerReady, d.instance.Generation,
+		metav1.ConditionFalse, rmn.ReasonNotStarted,
+		fmt.Sprintf("Started relocation to cluster %q", preferredCluster))
```

Comment:
The reason "ReasonNotStarted" contradicts the test "Started relocation".

Reply:
We could potentially reword this to "Relocation in progress to cluster %q"